content/docs/releases/v1.1.0.md

```diff
@@ -0,0 +1,84 @@
+
+As the number and variety of NSM use cases has grown, some use cases have emerged in which the Forwarder needs to do more complex cross connects than just P2P.  Point to multipoint (P2MP), by contrast to P2P, have a large number of mutually incompatible implementations.  Often P2MP will be spoken of in shorthand as if its is a single unitary thing: it is not.
+
+In order to support these sorts of use cases, NSM made a slight architectual enhancement: have the forwarder select the NSE (or in the case of P2MP NSEs) to which it should be connecting.  This change allows a Forwarder to support zero or more possible P2MP senarios.
+
+Given a Network Service that requires some particular P2MP capabilty (or any other kind of capability) from its Forwarder for one of its logical links, it is necessary to have some capacity of 'hinting' that requirement to the NSMgr so that it selects the correct Forwarder in the event that more than on is present.  NetworkService matches can now be decorated with 'metadata' with 'labels' for use by the NSMgr in selecting the correct Forwarder for each logical hop the match represents.
+
+
+### Interdomain improvements
+
+- [Network Service Mesh Interdomain](http://localhost:1313/docs/concepts/architecture/#inter-domain) now supports Spire Federation.   Spire Federation allows workloads from
+different Trust Domains to connect to a Network Service securely and subject to authorization.
+- [Network Service Mesh Interdomain CI](https://github.com/networkservicemesh/integration-interdomain-k8s) now tests interdomain simultaneously across:
+    - GKE
+    - EKS
+    - AKS
+    - Vanilla K8s on Bare Metal (using Equinix Metal)
+   validating that, for example, a Pod in GKE can connect to a Network Service being provided by a Pod in AKS.
+
+### VPP Forwarder Improvements
+
+- SRIOV support incorporated into the [VPP Forwarder](https://github.com/networkservicemesh/cmd-forwarder-vpp)
+  Pods can now request either a 'kernel' or 'vfio' interface backed by an SRIOV VF that connects it to its requested Network Service be injected into the Pod
+- Vlan remote mechanism: the [VPP Forwarder](https://github.com/networkservicemesh/cmd-forwarder-vpp) now supports use of vlan as a remote mechanism.
+  This allows the physical network to provide a Network Service, and distinguish between vWires by vlan tag.  Please note this feature is alpha as CI driven examples are not yet available.
+
+
+### OVS Forwarder
+A new forwarder based on the OpenvSwitch (OVS) data plane has been added to NSM as alternative to the VPP forwarder. The OVS forwarder can install its own OVS instance or re-use an existing OVS instance on the host. For standard use cases, the OVS forwarder supports the local kernel mechanism (using veth pairs) and remote VXLAN mechanism with typical kernel datapath performance.
```

Comment:
Can you also refer to ovs forwarder repo link https://github.com/networkservicemesh/cmd-forwarder-ovs ?

Reply:
@pperiyasamy Does this work: https://github.com/networkservicemesh/site/pull/207/commits/fc8a5c610e0ab7481945bc73eabe2ae5f9269212